workflow/environments/org/nesi/create_env_maui.sh

```diff
@@ -49,6 +52,20 @@ pip install --upgrade pip
 # Using xargs means that each package is installed individually, which
 # means that if there is an error (i.e. can't find qcore), then the other
 # packages are still installed. However, this is slower.
-xargs -n 1 -a ${env_path}/workflow/install_workflow/maui_python3_requirements.txt pip install -U
+xargs -n 1 -a ${env_path}/workflow/workflow/environments/org/nesi/maui_python3_requirements.txt pip install -U
+
+#source "${env_path}/workflow/workflow/environments//create_env_common_post.sh"
+
+for pkg in "${inhouse_pkgs[@]}";
+do
+    cd ${env_path}/${pkg}
+    pip install -U -r requirements.txt
```

Comment:
I'd prefer the relevant requirements to be handled in the requirements file above rather than each package requirements. Thoughts?

Reply:
Each package can have its own requirements - having all requirements into a single file means while each package is evolving (and has a new entry to its own requirements.txt), we need to keep maui_python3_requirements.txt up-to-date. I am not sure if it s worth extra overhead.